airflow/api_connexion/openapi/v1.yaml

```diff
@@ -134,6 +134,7 @@ info:
     | v2.0 | Initial release |
     | v2.0.2    | Added /plugins endpoint |
     | v2.1 | New providers endpoint |
+    | v2.3.0 | New mapped task instance endpoint |
```

Comment:
Looks like we should remove this summary of changes since we now have the lines like `*New in version 2.3.0*`. Not only the one you added but the whole of it

Reply:
Or we should update the summary to be more detailed and accurate?

Reply:
Problem I see is how easy it is to forget to update it every time we make changes. And a pre-commit for it is almost impossible (not sure)

Reply:
👍